# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-display/display-contents-dynamic-fieldset-legend-001-ref.html
# Commit: 867abe6c80a3
# Full Hash: 867abe6c80a36d66fec92f276c3a7d7c315637bf
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-03-01 09:36:12
# Description:
#   Bug 1694459 - Deal with 'display:contents' style changes on <fieldset> descendants better where it affects which <legend> is the rendered legend.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106566
# ==============================================================================

diff -r 77247863a2ea -r 867abe6c80a3 testing/web-platform/tests/css/css-display/display-contents-dynamic-fieldset-legend-001-ref.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/css/css-display/display-contents-dynamic-fieldset-legend-001-ref.html	Mon Mar 01 03:42:41 2021 +0000
@@ -0,0 +1,74 @@
+<!doctype html>
+<title>Reference: display:contents on fieldset/legend</title>
+<link rel="author" title="Mats Palmgren" href="https://bugzilla.mozilla.org/show_bug.cgi?id=1694459">
+<style>
+  body { font-size: 10px; }
+  fieldset {
+    border:1px solid;
+    padding: 1px 0;
+  }
+</style>
+<fieldset>
+<fieldset id='test1'>
+<legend>LEGEND</legend>
+</fieldset>
+</fieldset>
+
+<fieldset>
+<fieldset id='test2'>
+<legend>LEGEND</legend>
+</fieldset>
+</fieldset>
+
+<fieldset>
+<legend>OUTER LEGEND</legend>
+<fieldset id='test3'>
+<legend>INNER LEGEND</legend>
+</fieldset>
+</fieldset>
+
+<fieldset>
+<legend id='test4'>LEGEND</legend>
+<legend>NOT LEGEND</legend>
+</fieldset>
+
+<fieldset>
+<legend>LEGEND</legend>
+<legend id='test5'>NOT LEGEND</legend>
+</fieldset>
+
+<fieldset>
+<legend id='test6'>LEGEND</legend>
+<legend>NOT LEGEND</legend>
+</fieldset>
+
+<fieldset>
+<legend>LEGEND</legend>
+<legend>NOT LEGEND</legend>
+</fieldset>
+
+<fieldset>
+<legend>LEGEND</legend>
+</fieldset>
+
+<fieldset>
+<div id='test9'><legend>NOT LEGEND</legend></div>
+</fieldset>
+
+<fieldset>
+<legend>LEGEND</legend>
+<div>NOT LEGEND</div>
+</fieldset>
+
+<fieldset id='test11'>
+<legend>LEGEND</legend>
+<legend>NOT LEGEND</legend>
+</fieldset>
+
+<fieldset>
+<legend id='test12'>LEGEND</legend>
+</fieldset>
+
+<fieldset>
+<legend id='test13'>LEGEND</legend>
+</fieldset>